client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo/useReachContact.ts

```diff
@@ -57,6 +59,6 @@ export default useReachContact;
 interface Props {
     saveContact: (interactedContact: InteractedContact) => boolean;
     parsePerson: (person: GroupedInteractedContact, index: number) => InteractedContact;
-    formValues: InteractedContact;
+    formValues: GroupedInteractedContact;
```

Comment:
Are you sure its Grouped? Isn't the form just Interacted Contact?

Reply:
type script accepting to get in parse person variable from type Grouped so yes

Reply:
Ah right , cool